index.js

```diff
@@ -40,7 +40,12 @@ module.exports = function (source) {
 
   if (plugins) {
     plugins.forEach(function (plugin) {
-      parser.use(plugin);
+		 if(plugin.push){
+			 // allow array of options to be passed
+			 parser.use.apply(parser, plugin);
+		 }else{
```

Comment:
Can you leave spaces around the `if` and `else` clauses, like in the rest of the file?